frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -1035,7 +1035,6 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
               <AcmEmptyState
                 title={t('No results found')}
                 message={t('No results match the filter criteria. Clear filters to show results.')}
-                showIcon={false}
```

Comment:
Why did we remove this? I believe this is used when you filter the table down to no results, and currently we don't show an icon. If we do show an icon, I think the magnifying glass would be better here.

Reply:
It will default to the cubes icon, but yeah, I agree. Magnifying glass is a better fit...